syntheticsclientv2/integration_test.go

```diff
@@ -191,366 +194,579 @@ func TestLiveGetDevicesV2(t *testing.T) {
 
 }
 
-func TestLiveCreateHttpCheckV2(t *testing.T) {
+func TestLiveHttpCheckCreateUpdateDeleteV2(t *testing.T) {
```

Comment:
End to end test of Http Check:
1. Create
2. Update
3. Get
4. Update with original payload
5. Get
6. Delete